inference-engine/thirdparty/clDNN/kernel_selector/common/tensor_type.h

```diff
@@ -455,7 +569,13 @@ struct TensorBaseT : public TensorBase {
         if (same) {
             for (size_t i = 0; i < dims.size(); i++) {
                 same &= dims[i].v == t.dims[i].v && dims[i].pad.before == t.dims[i].pad.before &&
-                        dims[i].pad.after == t.dims[i].pad.after && dims[i].pitch == t.dims[i].pitch;
+                        dims[i].pad.after == t.dims[i].pad.after && dims[i].pitches.size() == t.dims[i].pitches.size();
+                if (same) {
+                    for (size_t p = 0; p < dims[i].pitches.size(); p++) {
+                        same &= dims[i].pitches[i].first == t.dims[i].pitches[i].first &&
+                                dims[i].pitches[i].second == dims[i].pitches[i].second;
```

Comment:
I suppose you shoud change one of "dims[i].pitches[i].second" on something else.